### PR TITLE
Add jupyter-sshd-proxy

### PR DESCRIPTION
**Pull request by jbusecke**
Over at LEAP we have been testing https://github.com/yuvipanda/jupyter-sshd-proxy (thanks @yuvipanda )and it works really nicely to connect to cloud hubs e.g. using vscode. I think this is a pretty broadly requested functionality and thus would be a good addition here.

Not sure about the pip dependency in this file. Should live somewhere else?

cc @dhruvbalwada

### CODE REVIEW

File: pangeo-notebook/environment.yml [hunk 1]
```diff
@@ -81,6 +81,9 @@ dependencies:
  - panel
  - parcels
  - param
+ - pip
+ - pip:
+     jupyter-sshd-proxy
```
[the remark this answers is not in the document]
Comment (jbusecke) [reply pangeo-notebook/environment.yml]: Switched the dependency!

**Review by weiji14 — APPROVED** [review]
Thanks @jbusecke! I'll merge this in today, but will get @scottyhq to tag a new version (possibly after #584 is merged too).

**Comment by jbusecke** [comment]
Thanks folks! Appreciate the quick turnaround!

**Pull request merged**